velox/functions/sparksql/DateTimeFunctions.h

```diff
@@ -482,6 +482,36 @@ struct DateFromUnixDateFunction {
   }
 };
 
+template <typename T>
+struct DateTruncFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  const tz::TimeZone* timeZone_ = nullptr;
```

Comment:
Please move the class member to the last of the struct as FromUnixtimeFunction.
And move it to private scope